Assets/Scripts/DataController.cs

```diff
@@ -145,7 +151,7 @@ void OnSceneLoaded(Scene scene, LoadSceneMode mode)
         runSimulation = false;
         if (scene.name == "Horizon")
         {
-            UpdateOnSceneLoad(true, 1, 100, .5f, false, true);
+            UpdateOnSceneLoad(true, 1, 100, .4f, false, true);
```

Comment:
Might want to change to `UpdateOnSceneLoad(true, 1, 100, .4f, true, true);` to show the constellations if we load the horizon scene from another scene.  I'll add the controls for selection in my upcoming PR).